Makefile

```diff
@@ -9,8 +9,20 @@ SECURITY_USER_PASSWORD := $(or $(SECURITY_USER_PASSWORD), pass)
 EDEN_EXEC=eden --client user --client-secret pass --url http://127.0.0.1:8080
 SERVICE_NAME=aws-eks-service
 PLAN_NAME=raw
-CLOUD_PROVISION_PARAMS=$(shell cat examples.json |jq '.[] | select(.service_name | contains("${SERVICENAME}")) | .provision_params')
-CLOUD_BIND_PARAMS=$(shell cat examples.json |jq '.[] | select(.service_name | contains("${SERVICENAME}")) | .bind_params')
+
+# Use the env var INSTANCE_NAME for the name of the instance to be created, or
+# "instance-$USER" if it was not specified. 
+#
+# We do this to minimize the chance of people stomping on each other when
+# provisioning resources into a shared account, and to make it easy to recognize
+# who resources belong to.
+#
+# We can also use a job ID during CI to avoid collisions from parallel
+# invocations, and make it obvious which resources correspond to which CI run.
+INSTANCE_NAME ?= instance-$(USER)
+
+CLOUD_PROVISION_PARAMS="{ \"cluster_name\": \"${INSTANCE_NAME}\" }"
```

Comment:
Shouldn't this be the make variable $(INSTANCE_NAME) to pick up the default?

Reply:
We confirmed that this works as-is, but I'll send up a PR later that makes it consistent.